conda-recipe/meta.yaml

```diff
@@ -7,7 +7,7 @@ build:
 
   script:
     - cd $RECIPE_DIR/..
-    - $PYTHON setup.py install
+    - LDFLAGS="-shared" FFLAGS=-fPIC $PYTHON setup.py install # [not win]
```

Comment:
I've removed this and the setup.py mods from your branch [here](https://github.com/roryyorke/Slycot/commit/813c6942a402176649c7aecc640b4f061b7e7e73); passing Travis CI results [here](https://travis-ci.org/roryyorke/Slycot/builds/241721782).